README.md

```diff
@@ -210,6 +210,39 @@ Command: <code>python3 -m pip install --user --upgrade pip</code>
 	Link to Solution: <a href="https://github.com/Udayraj123/OMRChecker/issues/70#issuecomment-1268094136">#70</a>
 </details>
 
+### Running in a Container Runtime (Docker or Podman)
+
+The repo contains a containerfile that can be used to run ORM checker without 
+interfering with a system python installation.
```

Comment:
Let's make the advantage more explicit mention that we can run the container in any exam directory without having to keep the code locally. On that matter, we can consider publishing the image to dockerhub as well so as to skip the build step for those who wish to directly try it out.